tests/test_terminal.py

```diff
@@ -0,0 +1,68 @@
+import shutil
+import pytest
+import json
+import asyncio
+
+
+@pytest.fixture
+def terminal_path(tmp_path):
+    subdir = tmp_path.joinpath('terminal_path')
+    subdir.mkdir()
+
+    yield subdir
+
+    shutil.rmtree(str(subdir), ignore_errors=True)
+
+
+async def test_terminal_create(fetch):
+    await fetch(
+        'api', 'terminals',
+        method='POST',
+        # body=body,
+        allow_nonstandard_methods=True,
+    )
+
+    resp_list = await fetch(
+        'api', 'terminals',
+        method='GET',
+        # body=body,
+        allow_nonstandard_methods=True,
+    )
+
+    data = json.loads(resp_list.body.decode())
+
+    assert len(data) == 1
+
+
+async def test_terminal_create_with_cwd(fetch, ws_fetch, terminal_path):
+    resp = await fetch(
+        'api', 'terminals',
+        method='POST',
+        body=json.dumps({'cwd': terminal_path.as_posix()}),
```

Comment:
I think we'll have to skip these tests on Windows, since `cwd` won't work on a Windows prompt.